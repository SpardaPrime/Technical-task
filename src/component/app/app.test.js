import renderer from "react-test-renderer";
import App from "./app";

test("snapshot: component App", () => {
  const component = renderer.create(<App />).toJSON();
  expect(component).toMatchSnapshot();
});
