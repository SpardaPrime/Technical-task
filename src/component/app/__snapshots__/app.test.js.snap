// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot: component App 1`] = `
Array [
  <div
    className="m"
  >
    <h1
      className="h1-title"
    >
      Employees
    </h1>
    <div
      className="eploy-block"
    >
      <h2>
        Loading employees
      </h2>
    </div>
  </div>,
  <div
    className="b-block"
  >
    <h1
      className="h1-title"
    >
      Employees birthday
    </h1>
    <div
      className="n-select"
    >
      <h2>
        No selected employees
      </h2>
    </div>
  </div>,
]
`;
